chia/data_layer/data_layer_client.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+import aiosqlite
+import aiohttp
+import time
+from chia.data_layer.data_store import DataStore
+from chia.util.db_wrapper import DBWrapper
+from chia.types.blockchain_format.tree_hash import bytes32
+from chia.types.blockchain_format.program import Program
+from chia.util.byte_types import hexstr_to_bytes
+from chia.data_layer.data_layer_types import Status, NodeType
+from typing import List, Any, Dict
+
+
+class DataLayerClient:
+    async def init_db(self) -> None:
+        self.db_connection = await aiosqlite.connect(":memory_client:")
+        self.db_wrapper = DBWrapper(self.db_connection)
+        self.data_store = await DataStore.create(db_wrapper=self.db_wrapper)
+        tree_id = bytes32(b"\0" * 32)
+        await self.data_store.create_tree(tree_id=tree_id)
+
+    async def download_data_layer(self) -> None:
+        await self.init_db()
+        async with aiohttp.ClientSession() as session:
+            verbose = False
+            tree_id = "0x0000000000000000000000000000000000000000000000000000000000000000"
+            url = f"http://0.0.0.0:8080/get_tree_root?tree_id={tree_id}"
+            async with session.get(url) as r:
+                root_json = await r.json()
+            node = root_json["node_hash"]
+            root_hash = root_json["node_hash"]
```

Comment:
did you mean to look up the "root_hash" key here?

Reply:
no, this is how it's stored in the DB:
https://github.com/Chia-Network/chia-blockchain/blob/atari/chia/data_layer/data_store.py#L72

Reply:
then I would propose you simplify this (and make the intention clear):

```
node = root_hash = root_json["node_hash"]
```